src/transformStateWithClones.js

```diff
@@ -7,7 +7,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let cloneState = { ...state };
+  const result = [];
```

Comment:
Don't use names like result in your code, it's a bad practice, something like stateHistory would be a perfect fit